### PR TITLE
Add information on querying Ref Tables in Logs

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?

- Add section in Logs Analytics about using Reference table as query filters in Logs.

### Motivation

[DOCS-5800](https://datadoghq.atlassian.net/browse/DOCS-5800)

### Additional Notes

Ready for docs review, do not merge please.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5800]: https://datadoghq.atlassian.net/browse/DOCS-5800?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/logs/explorer/analytics/_index.md [hunk 1]
```diff
@@ -99,10 +102,20 @@ Apply a formula on one or multiple queries by clicking on the `+ Add` button nex
...
 To apply formulas with multiple queries, all queries must be grouped by the same facet. In the example above, both queries are grouped by `Webstore Store Name`.
 
-You can apply a function to a formula by clicking on the `Σ` icon. Here is an example of how to apply a [Timeshift function][10] on the proportion of error logs in all logs to compare current data with data from one week before:
+You can apply a function to a formula by clicking on the `Σ` icon. Here is an example of how to apply a [Timeshift function][12] on the proportion of error logs in all logs to compare current data with data from one week before:
 
 {{< img src="logs/explorer/group/timeshift_function_logs.jpg" alt="The query editor showing a formula with the week before timeshift function applied to it" style="width:100%;" >}}
 
+## Filter logs based on Reference Tables
+
+<div class="alert alert-warning">Reference Tables and the feature to filter logs based on Reference Tables are in beta. There are no billing implications for defining and querying Reference Tables. There is a limit of 100 Reference Tables per account and a limit of 1000 rows in a Reference Table.</div>
```
Comment (siddhingra) [inline content/en/logs/explorer/analytics/_index.md:111]: There may be costs to defining a reference table in the future so let's leave that part out.

Comment (estherk15) [reply content/en/logs/explorer/analytics/_index.md:111]:
```suggestion
<div class="alert alert-warning">Reference Tables and the feature to filter logs based on Reference Tables are in beta. There is a limit of 100 Reference Tables per account and a limit of 1000 rows in a Reference Table.</div>
```

File: content/en/logs/explorer/analytics/_index.md [hunk 2]
```diff
@@ -99,10 +102,20 @@ Apply a formula on one or multiple queries by clicking on the `+ Add` button nex
...
 
 {{< img src="logs/explorer/group/timeshift_function_logs.jpg" alt="The query editor showing a formula with the week before timeshift function applied to it" style="width:100%;" >}}
 
+## Filter logs based on Reference Tables
+
+<div class="alert alert-warning">Reference Tables and the feature to filter logs based on Reference Tables are in beta. There is a limit of 100 Reference Tables per account and a limit of 1000 rows in a Reference Table.</div>
+
+Reference Tables allow you to combine metadata with logs, providing more information to resolve application issues. Add a query filter based on a Reference Table to perform lookup queries. For more information on creating and managing this feature, see the [Reference Tables][17] guide.
+
+To apply a query filter with Reference Tables, click on the `+ Add` button next to the query editor and select **Reference Table**. In the following example, the Reference Table query filter is used to search all recent logs that include a malicious IP address from a thread intel reference table:
```
Comment (buraizu) [inline content/en/logs/explorer/analytics/_index.md:115]:
```suggestion
To apply a query filter with Reference Tables, click on the `+ Add` button next to the query editor and select **Reference Table**. In the following example, the Reference Table query filter is used to search all recent logs that include a malicious IP address from a threat intel reference table:
```

File: content/en/logs/explorer/analytics/_index.md [hunk 3]
```diff
@@ -99,10 +102,20 @@ Apply a formula on one or multiple queries by clicking on the `+ Add` button nex
...
+
+{{< img src="/logs/explorer/group/reference_tables_logs.png" alt="Use Reference Tables in the Log Explorer" border="true" popup="true" style="width:100%;" >}}
```
Comment (buraizu) [inline content/en/logs/explorer/analytics/_index.md:117]:
```suggestion
{{< img src="/logs/explorer/group/reference_tables_logs.png" alt="The Datadog log explorer with reference table search options highlighted" border="true" popup="true" style="width:100%;" >}}
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Just flagging what might be a typo, and made a suggestion for the alt text

**Comment by estherk15** [comment]
Thanks for the review @buraizu!

**Review by buraizu — APPROVED** [review]

**Pull request merged**